src/main/java/engine/property/manager/PropertyManager.java

```diff
@@ -44,6 +44,7 @@ public static void setProperty(String systemProperty, String systemStringValue)
 
     public static BrowserName getBrowserName() {
         return BrowserName.valueOf(getProperty(PropertyKeys.BROWSER_NAME));
+
```

Comment:
Accidental change probably, to be removed:)

Reply:
resolved